q2_feature_classifier/_taxonomic_classifier.py

```diff
@@ -95,7 +95,7 @@ def _2(data: Pipeline) -> TaxonomicClassiferTemporaryPickleDirFmt:
 
 
 @plugin.register_transformer
-def _3(dirfmt: TaxonomicClassifierDirFmt) -> dict:
+def _3(dirfmt: TaxonomicClassifierDirFmt) -> Pipeline:
```

Comment:
I'll take your authority on this one, because my understanding of the transformer architecture is superficial. However, the transformer used to return a `dict`, and this will never return a `Pipeline` because it will raise a `ValueError`. As I said however, you're the experts.